src/solver/ts-generator/availability.cpp

```diff
@@ -198,13 +178,16 @@ int AvailabilityTSgenerator::durationGenerator(Data::StatisticalLaw law,
     return 0;
 }
 
-void AvailabilityTSgenerator::run(AvailabilityTSGeneratorData& tsGenerationData) const
+Matrix<double> AvailabilityTSgenerator::run(AvailabilityTSGeneratorData& tsGenerationData) const
```

Comment:
**AvailabilityTSgenerator::run(...)** now returns the generated TS, and no longer take the generated TS as argument.
It's more natural and clear like this, and allows to simplify it's calls (for example, see **generateLinkTimeSeries(...)** below)